src/Mcfedr/AwsPushBundle/Message/Message.php

```diff
@@ -608,6 +642,7 @@ private function getGcmJson()
             'collapse_key' => $this->collapseKey,
             'time_to_live' => $this->ttl,
             'delay_while_idle' => $this->delayWhileIdle,
+            'priority' => $this->priority ? $this->priority : self::PRIORITY_HIGH,
```

Comment:
Maybe dont add the key unless the user has set it? or set $this->priority to be high by default

Reply:
this line makes high priority as a default one unless other is set, doen't it? 
or is it better to set the value when I create $priority field e.g.
private $priority = self::PRIORITY_HIGH;
?

Reply:
yea, I'd set it in the field and avoid having to deal with `null`